internal/venice-test-common/src/integrationtest/java/com/linkedin/venice/integration/utils/VeniceServerWrapper.java

```diff
@@ -225,28 +229,38 @@ static StatefulServiceProvider<VeniceServerWrapper> generateService(
       // generate the kafka cluster map in config directory
       VeniceConfigLoader.storeKafkaClusterMap(configDirectory, kafkaClusterMap);
 
-      if (!forkServer) {
-        VeniceConfigLoader veniceConfigLoader =
-            VeniceConfigLoader.loadFromConfigDirectory(configDirectory.getAbsolutePath());
+      String d2ServiceName = D2TestUtils.getRouterD2ServiceName(null, clusterName);
 
+      VeniceProperties serverProperties = serverPropsBuilder.build();
+      boolean https = serverProperties.getBoolean(ROUTER_HTTP2_INBOUND_ENABLED, false);
```

Comment:
If we're only using this config to decide if servers should support SSL, then we should use `SERVER_HTTP2_INBOUND_ENABLED`